Makefile

```diff
@@ -39,8 +39,12 @@ ISORT_CMD := $(PIPENV_RUN) isort
 NOSE_CMD := $(PIPENV_RUN) nose2
 PYLINT_CMD := $(PIPENV_RUN) pylint
 
+# AWS variables
+AWS_DEFAULT_REGION = eu-central-1
+
 # Docker variables
-DOCKER_IMG_LOCAL_TAG = swisstopo/$(SERVICE_NAME):local
+DOCKER_REGISTRY = 974517877189.dkr.ecr.eu-central-1.amazonaws.com
+DOCKER_IMG_LOCAL_TAG = $(DOCKER_REGISTRY)/$(SERVICE_NAME):local
```

Comment:
I would here use the similar local tag as the other projects, `$(SERVICE_NAME):local-$(USER)` optionaly add also the git short hash.